domain-ee/ee-ep-merge-app/src/python_src/config.py

```diff
@@ -81,4 +81,5 @@ class ClientName(str, Enum):
 POSTGRES_PORT = os.environ.get("POSTGRES_PORT") or "5432"
 POSTGRES_DB = os.environ.get("POSTGRES_DB") or "vro"
 POSTGRES_SCHEMA = os.environ.get("POSTGRES_SCHEMA") or "claims"
-SQLALCHEMY_DATABASE_URI = f"postgresql://{POSTGRES_USER}:{POSTGRES_PASSWORD}@{POSTGRES_HOST}:{POSTGRES_PORT}/{POSTGRES_DB}"
+POSTGRES_URL = os.environ.get("POSTGRES_URL") or f"postgresql://{POSTGRES_HOST}:{POSTGRES_PORT}/{POSTGRES_DB}"
+SQLALCHEMY_DATABASE_URI = POSTGRES_URL.replace('postgresql://', f"postgresql://{POSTGRES_USER}:{POSTGRES_PASSWORD}@")
```

Comment:
I suggest checking that `POSTGRES_USER` is set before doing this replacement. It's possible that the credentials could someday get moved into `POSTGRES_URL` which would cause an error here.

Reply:
Ah. It should be set to `vro_user` by default if the env variable doesn't exist. Are you suggesting removing that default value and then make the check to see if its not None?

Reply:
I think Sam is saying, the env var `$POSTGRES_URL` itself could already contain the user and password, in which case this line would produce an unusable value for `SQLALCHEMY_DATABASE_URI`. Doesn't matter what the default value is.

Easiest way is to let Python's `urllib.parse.urlparse` do the dirty work of string splitting.